index.html

```diff
@@ -446,7 +446,9 @@
           <blockquote class="informative">
             Non-normative note: In the presence of a <code>Want-Digest</code> header, the current <code>Digest</code>
             value should be calculated to enable fixity verification. Otherwise, it is expected that ETag values may be
-            cached. See <a href="#persistence-fixity"></a>.
+            cached. Clients should use the <code>Cache-Control: no-cache</code> header (see [[RFC7234]]
```

Comment:
Would it be safer to replace `Clients should use` to `Clients are advised to use`?

Reply:
I'm not sure about this because it is the designers of clients that are advised

Reply:
To be clear, are you suggesting that the current wording expresses the intent to your liking, @zimeon ?

Reply:
Yes, I think the current wording is adequate (and I think it would not be quite right to say "clients are advised")